mobile/sensors.html

```diff
@@ -40,7 +40,7 @@ <h2>Technologies in progress</h2>
 
         <p>The <a href="https://www.w3.org/TR/motion-sensors/">Motion Sensors Explainer</a> document is an introduction to low-level and high-level motion sensors, their relationship, inner workings and common use-cases.</p>
 
-        <p data-feature="Geolocation">The <a data-featureid="geolocation-sensor">Geolocation Sensor</a> is an API for obtaining geolocation reading from the hosting device. The feature set of the Geolocation Sensor is similar to that of the Geolocation API, but it is surfaced through the Generic Sensor API, improves security and privacy, and is extensible.</p>
+        <p data-feature="Geolocation">The <a data-featureid="geolocation-sensor">Geolocation Sensor</a> is an API for obtaining geolocation reading from the hosting device. The feature set of the Geolocation Sensor is similar to that of the Geolocation API, but it is surfaced through the Generic Sensor API, allows background operations, improves security and privacy, and is extensible.</p>
```

Comment:
I think it's worth mentioning slightly more concrete use cases as "background operations" does not mean much in itself. I suggest:

s/allows background operations/allows continuous geotracking and geofencing (one-time alerts when a user crosses a specific area) operations in the background